GameFinder/AStoreHandler.cs

```diff
@@ -25,6 +25,6 @@ public abstract class AStoreHandler<TGame> where TGame : AStoreGame
         /// Find all games
         /// </summary>
         /// <returns></returns>
-        public abstract bool FindAllGames();
+        public abstract Result<bool> FindAllGames();
```

Comment:
Had the idea that perhaps this interface could just be `Result<List<TGame>> FindAllGames`?  (Remove the Games member)

With Result, it just fits in real nicely to just give it straightaway, I feel.  If someone is interested if there was issues, they could just check if the result has errors